lib/aullar/flair.moon

```diff
@@ -35,8 +35,22 @@ set_line_type_from_flair = (cr, flair) ->
       cr.dash = {6, 3}
 
 draw_ops = {
-  rectangle: (flair, x, y, width, height, cr) ->
+  custom: (flair, x, y, width, height, cr) ->
+    custom_fn = flair.custom_draw
+    if custom_fn and pcall custom_fn, flair, x, y, width, height, cr
```

Comment:
Should be all good now.
I think it's worth it to catch the error here, otherwise the whole editor draw is wiped if it's an early drawn flair.
I imagine a custom-draw is something a theme-maker "improvises forward", so there's likely a few errors in the progress.